apps/webstore-e2e/src/integration/pages/products/quotable/baby-sign.spec.ts

```diff
@@ -0,0 +1,63 @@
+import { QuotableDesignEnum } from '@assets';
+import { DesignFontEnum, DesignTypeEnum, IDesign, IQoutable } from '@interfaces';
+import { CookieStatus, LocalStorageVars } from '@models';
+
+const quotableDesing: IQoutable = {
+  font: DesignFontEnum.archaMedium,
+  fontSize: 40,
+  designSrc: QuotableDesignEnum.frame18,
+  text: 'skrt skrt',
+};
+const mockQuotableDesign: IDesign = {
+  designId: 'MOCK_ID',
+  designProperties: quotableDesing,
+  mutable: true,
+  user: null,
+  designType: DesignTypeEnum.quotable,
+};
+
+describe('BabySignProductPage', () => {
+  beforeEach(() => {
+    localStorage.setItem(LocalStorageVars.cookiesAccepted, `"${CookieStatus.accepted}"`);
+    localStorage.setItem(LocalStorageVars.firstVisit, 'true');
+    cy.visit('/products/quotable?productType=BABY_SIGN');
+  });
+
+  describe('Baby sign product page', () => {
+    it('has the right frames', () => {
+
+    });
+
+    it('goes to the right frame when pressing prev', () => {
```

Comment:
The wording of this is confusing. 'it goes to the correct frame' would make more sense in my opinion, since right implies a direction, and given that there's an arrow button for the design type on the right of the modal display, it would make more sense to say correct frame.